src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbContent/BreadcrumbContent.jsx

```diff
@@ -22,7 +22,13 @@ export const BreadcrumbContent = forwardRef(
     if (isClickable && (link || onClick)) {
       if (link) {
         return (
-          <a className={className} href={link} onKeyDown={onKeyDown} aria-current={isCurrent ? "page" : undefined}>
+          <a
+            className={className}
+            href={link}
+            onKeyDown={onKeyDown}
+            aria-current={isCurrent ? "page" : undefined}
+            role="link"
```

Comment:
when element is a native link (a) there is no need to add also role link

Reply:
so let's keep the button role and remove this one

Reply:
Done

Reply:
Done